kolibri/plugins/learn/assets/src/views/HybridLearningCardGrid.vue

```diff
@@ -79,7 +79,7 @@
       :isLeaf="content.is_leaf"
       :progress="content.progress_fraction || 0"
       :numCoachContents="content.num_coach_contents"
-      :link="genContentLink(content.id, content.is_leaf)"
+      :link="genContentLink(content.id, content.is_leaf, backRoute)"
```

Comment:
Why is the `genContentLink` above passed the context, but this one isn't?

Reply:
It's an optional param. i.e. library page does not need it, but since the lessons cards _do_ since their route requires a specific classid and lessonid, it is passed there. Maybe I can name it something clearer...

Reply:
I think would be good to just always pass it for consistency - means that we won't get bitten in the behind in future when we decide we need to pass context from the library page (like the search term that someone arrived at the content from) and we can't understand why the context is missing.